cli.js

```diff
@@ -23,6 +23,7 @@ program
 program
   .command('start')
   .description('start streaming video from the raspberry pi camera module')
+  .option('-n, --nopreview', 'Removes the preview screen')
```

Comment:
Shouldn't be a preview screen at all, so an option isn't necessary.

Reply:
On the Pi 3, I keep getting the preview screen opening up and taking over the screen, preventing any interaction with the Pi. Only solution is to restart the Pi. This flag allows me to prevent the preview from opening up in the first place.

Reply:
> On the Pi 3, I keep getting the preview screen opening up and taking over the screen, preventing any interaction with the Pi.

Right. raspi-live shouldn't be opening a preview screen at all and no one should want it to, so the option isn't needed.

Reply:
Moved your comment from the main PR body back to this thread:
> raspy-live is the package that IS opening the preview by default, so I had no choice but to add the option and use it.

You don't need to add this option to raspi-live, just to the list of options given to raspivid in `server.js` like you did below. Adding the option in `cli.js` doesn't actually do anything. If you were to remove just this line, nothing would change.